packages/enzyme/src/Utils.js

```diff
@@ -362,3 +362,15 @@ export function displayNameOfNode(node) {
 
   return type.displayName || (typeof type === 'function' ? functionName(type) : type.name || type);
 }
+
+export function sym(s) {
+  return typeof Symbol === 'function' ? Symbol.for(`enzyme.${s}`) : s;
+}
+
+export function privateSet(obj, prop, value) {
+  Object.defineProperty(obj, prop, {
```

Comment:
no real reason, no. It would add some syntactic overhead relative to `obj[prop]`, but i'm not opposed to it if we want to. I figured we'd just keep this setup until private properties are at whatever stage makes you comfortable with us using them :)